### ISSUE TITLE
 errors=1, failures=4  from  _DL_DEVEL_ANNEX=1 mode on 0.11.x cron job

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#3472]
https://travis-ci.org/datalad/datalad/jobs/541363548
failures suggest upcoming change in git-annex behavior/output

```shell
$ http_proxy= PATH=$PWD/../tools/coverage-bin:$PATH $NOSE_WRAPPER `which nosetests` $NOSE_OPTS -A "$NOSE_SELECTION_OP($NOSE_SELECTION)" --with-doctest --with-cov --cover-package datalad --logging-level=INFO $TESTS_TO_PERFORM
S.........................S...S.......................................F....................................................................................................S......SS.....SSSSSSSS.......SSSS.SS..................................S......................................................LFA_Read: read failure
.LFA_Read: read failure
.........................S....................................S......S.....F..S..........E.F..................................................SS...............S.SSSSSS.................................S.............................SS.............................S..............F.........CommandError: command '['ssh', '-o', 'ControlPath="/home/travis/.cache/datalad/sockets/04ad7034"', 'localhost', 'export "PATH=/usr/lib/git-annex.linux:$PATH"; exit 42']' failed with exitcode 42
Failed to run ['ssh', '-o', 'ControlPath="/home/travis/.cache/datalad/sockets/04ad7034"', 'localhost', 'export "PATH=/usr/lib/git-annex.linux:$PATH"; exit 42'] under None. Exit code=42.
.......S..................................................................SS.SSSS.....................................S...S..............................................................
----------------------------------------------------------------------
Ran 1 test in 0.036s
OK
.......SS...................................................SS..S.............................................SS.....S......S
======================================================================
ERROR: datalad.support.tests.test_annexrepo.test_fake_dates
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 615, in newfunc
    return t(*(arg + (filename,)), **kw)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/support/tests/test_annexrepo.py", line 2374, in test_fake_dates
    ar.repo.git.cat_file("blob", "git-annex:uuid.log"))
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/git/cmd.py", line 548, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/git/cmd.py", line 1014, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/git/cmd.py", line 825, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('/usr/lib/git-annex.linux/git') failed due to: exit code(128)
  cmdline: /usr/lib/git-annex.linux/git cat-file blob git-annex:uuid.log
  stderr: 'fatal: Not a valid object name git-annex:uuid.log'
======================================================================
FAIL: datalad.distribution.tests.test_create_sibling.test_target_ssh_simple
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 869, in newfunc
    return func(*args, **kwargs)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 921, in newfunc
    return func(*args, **kwargs)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 791, in newfunc
    t(*(arg + (uri,)), **kw)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 615, in newfunc
    return t(*(arg + (filename,)), **kw)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 615, in newfunc
    return t(*(arg + (filename,)), **kw)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/distribution/tests/test_create_sibling.py", line 219, in test_target_ssh_simple
    ok_endswith(target_description, basename(target_path))
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 327, in ok_endswith
    msg="String %r doesn't end with %r" % (s, suffix))
AssertionError: String '' doesn't end with 'basic'
======================================================================
FAIL: datalad.support.tests.test_annexrepo.test_AnnexRepo_always_commit
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 615, in newfunc
    return t(*(arg + (filename,)), **kw)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/support/tests/test_annexrepo.py", line 731, in test_AnnexRepo_always_commit
    eq_(num_commits, 3)
AssertionError: 2 != 3
======================================================================
FAIL: datalad.support.tests.test_annexrepo.test_get_description
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 615, in newfunc
    return t(*(arg + (filename,)), **kw)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 615, in newfunc
    return t(*(arg + (filename,)), **kw)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/support/tests/test_annexrepo.py", line 1707, in test_get_description
    annex1_description + ' [annex1]')
AssertionError: 'annex1' != ' [annex1]'
======================================================================
FAIL: datalad.support.tests.test_repodates.test_check_dates
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 442, in newfunc
    return t(*(arg + (d,)), **kw)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/support/tests/test_repodates.py", line 68, in test_check_dates
    for x in results["older"].values()))
AssertionError: None
```

**Comment by kyleam** [comment]
Thanks for posting.  I'll take a look.

**Comment by kyleam** [comment]
test_annexrepo.test_get_description and test_create_sibling.test_target_ssh_simple should be due to <https://git-annex.branchable.com/bugs/annex_init_no_longer_generates_default_description/>.

**Comment by kyleam** [comment]
The same commit from the bug report above also leads to no uuid.log file being created, which id the source of the remaining test failures: <https://git-annex.branchable.com/bugs/annex_init_no_longer_generates_default_description/#comment-51726ef5384843c7c6c2534f4024eca5>

Let's see what comes out of that bug report.

**Comment by yarikoptic** [comment]
I was confused first since the a14f6ce75 change in git-annex came before my recent test build, and I was assuming that the recent test of the standalone git annex build succeeded.  BUT
https://github.com/datalad/datalad/pull/3466
actually failed and did show these failures!

**Comment by kyleam** [comment]
Fixed upstream in 84e729fda (fix init default description reversion, 2019-06-20).  Verified locally that these tests pass with that commit (well, except for test_target_ssh_simple, but that is clearly caused by the empty description regression).

**Comment by kyleam** [comment]
These pass in Travis build with latest git-annex (7.20190708).

https://travis-ci.org/datalad/datalad/builds/555949550
